packages/repo-tools/cli-report.md

```diff
@@ -39,6 +40,15 @@ Options:
   -h, --help
 ```
 
+### `backstage-repo-tools backend-lint`
```

Comment:
Kinda wondering if there's a better structure here, something like `backstage-repo-tools lint legacy-backend-exports` or so? Just leaving space for adding some more of these without too much clutter?

Reply:
I like that idea 👍